### PR TITLE
Update like_util.py Read media type of the post

### LINKED ISSUE #6144: Cannot detect post media type. Skip https://www.instagram.com
from instapy import InstaPy

session = InstaPy(username='xxxxxxxx', password='xxxxxxxx', headless_browser=True)
session.login()
session.set_action_delays(enabled=True, like=120)
session.set_user_interact(amount=3, randomize=True, percentage=100)
session.like_by_tags(['car'], amount=500, interact=True )
session.end()

### LINKED ISSUE #6243: Cannot detect post media type. skip
Hi. 
Im working on windows and python 3.9 and latest instapy version.
my code:

from instapy import InstaPy

session = InstaPy(username="<my user>", password="<my pass>")
session.login()
session.like_by_tags(["tag"], amount=1)

### PR DESCRIPTION
**Pull request by alokkumarsbg**
# Pull Request Template

## Description

Update line which read media type of post. Earlier the media type was reading as none

Do not include any personal data.

Fixes # (issue)
Will read media type from svg tag

## How Has This Been Tested?

Executing function like_by_tags to test the changes 

- [x] Test

## Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have checked my code and corrected any misspellings
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project, `black -t py34`
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by schealex — APPROVED** [review]
lgtm

File: instapy/browser.py [hunk 1]
```diff
@@ -158,7 +158,7 @@ def set_selenium_local_session(
 
 
 def proxy_authentication(browser, logger, proxy_username, proxy_password):
-    """ Authenticate proxy using popup alert window """
+    """Authenticate proxy using popup alert window"""
```
Comment (elulcao) [inline instapy/browser.py]: Why the change, is the space affecting functionality?
The PR is for updating the XPATH, isn't it?
`instapy/like_util.py`

**Comment by elulcao** [comment]
Travis-CI is complaining for some errors, could you take a look
thanks! :)

File: instapy/browser.py (hunk 1 above)
Comment (alokkumarsbg) [reply instapy/browser.py]: Space is not affecting functionality, but that space is causing Travis CI not accepting black -t py34 for code

We can ignore Travis CI error  for black -t py34. It doesn't make sense removing space from all main branch files

Comment (elulcao) [reply instapy/browser.py]: I see, it would be better to honor Travis results. So, thanks for taking care of this issue 😀

Comment (alokkumarsbg) [reply instapy/browser.py]: I will do the required changes. As running black -t py34 give no error while doing it on my server.

Let me keep doing changes and have to test that here.

**Comment by schealex** [comment]
@alokkumarsbg if you grant me permissions on your fork i can push the fix, have it done locally already just need to be able to push it.

**Comment by alokkumarsbg** [comment]
> @alokkumarsbg if you grant me permissions on your fork i can push the fix, have it done locally already just need to be able to push it.

Thanks @schealex . permission granted

File: instapy/browser.py (hunk 1 above)
Comment (schealex) [reply instapy/browser.py]: I've done it

**Review by schealex — APPROVED** [review]
should be fine now

**Review by schealex — APPROVED** [review]

**Comment by schealex** [comment]
@elulcao can you do the merge?

**Comment by elulcao** [comment]
Hello @schealex 
I merged the previuos commit, so could you rebase and Update the changelog file with info regarding this commit, Thanks :)

**Comment by alokkumarsbg** [comment]
> Hello @schealex
> I merged the previuos commit, so could you rebase and Update the changelog file with info regarding this commit, Thanks :)

@elulcao Changelog file updated

**Review by AwJHub — COMMENTED** [review]
still Cannot detect post media type.

**Comment by alokkumarsbg** [comment]
> still Cannot detect post media type.

@AwJHub Give me more details so i can test

**Comment by xerZV** [comment]
same problem here

**Comment by juancv3d** [comment]
I tried to fix but didn't work

**Review by dahch — APPROVED** [review]

**Review by dahch — APPROVED** [review]

**Comment by gdr2404** [comment]
Did anyone solve this problem?

**Comment by elulcao** [comment]
We need to verify the change proposed is solving the issue reported.

**Review by elulcao — APPROVED** [review]

**Comment by wuhei** [comment]
Hello, I don't really know how to do it otherwise but i believe that u7YqG div class has now changed to CzVzU, changing the class in like_util.py to CzVzU seems to fix the post Media type drama

**Comment by gdr2404** [comment]
Just tried that and it hasn't solved the issue for me :(

**Comment by kharazian** [comment]
I used develop branch of https://github.com/schealex/InstaPy.git and it has been working till today that I face same problem.

**Comment by bogdan-darkreign** [comment]
Hi  All ,

Modify like_util.py  > line 867  
from  :  + "/']/child::div[@class='u7YqG']/child::div/*[name()='svg']" 
to :    + "/']/child::div[@class='CzVzU']/child::div/*[name()='svg']"

Cheers

**Comment by KenanYaman** [comment]
> Hi All ,
> 
> Modify like_util.py > line 867 from : + "/']/child::div[@Class='u7YqG']/child::div/_[name()='svg']" to : + "/']/child::div[@Class='CzVzU']/child::div/_[name()='svg']"
> 
> Cheers

This worked. Thanks.

**Comment by gdr2404** [comment]
It must be something else for my set up. Still not working for me. tried the original branch and the one posted above by @kharazian 

Tried changing the line in util.py like @bogdan-darkreign & @KenanYaman have had success with but its still not working on my system. Feel like im gonna have to try and set this up on a mac and see if it works. It hasn't on either of my windows machines.

**Comment by kharazian** [comment]
> It must be something else for my set up. Still not working for me. tried the original branch and the one posted above by @kharazian
> 
> Tried changing the line in util.py like @bogdan-darkreign & @KenanYaman have had success with but its still not working on my system. Feel like im gonna have to try and set this up on a mac and see if it works. It hasn't on either of my windows machines.

I think it is another things. What is yuor exact error?

**Comment by kharazian** [comment]
> 134058995-e6351187-a8af-48d3-a41a-c3b18d2f552d
> 
> this is the error that i'm getting. And the last line just repeats.
> 
> It's logging in no bother, cookie pop up box appears and it gets past that and goes to my feed page where it hangs. After a little while it goes to my profile page only for a second or two before returning to my feed. From here it starts to cycle through the images in my feed but not liking them. Every now and then i get a "total links fetched for analysis: 0" ​
> 
> ​I'm using a older version of firefox (version 60)as recomended by other users and others have asked if instagram is blocking me, but it doesnt seem to be as i can stop the script while firefox is open and like the images manually. Im also able to use other scripts such as adding people and i'm not having any isuees with that. just the liking image.

changing the xpath from from //article/div[3]/div[2]/a to //article/div/div[3]/div/div/div[2]/a for get_links_from_feed item in xpath_compile.py solved the problem #6342

**Review by omarsameh — COMMENTED** [review]
This didn't fix the issue, Still getting "Cannot detect post media type. Skip ...."

**Review by omarsameh — COMMENTED** [review]
This didn't fix it
